setup.cfg

```diff
@@ -1,4 +1,7 @@
 [flake8]
-# 201 and 202 are spaces after ( and before ), 501 is line length. These
-# are exceptions allowed (encouraged?) by Galaxy style guidelines.
-ignore = E201,E202,E501
+# These are exceptions allowed (encouraged?) by Galaxy style guidelines.
+# 128 continuation line under-indented for visual indent
+# 201 and 202 are spaces after ( and before )
+# 203 whitespace before ':'
```

Comment:
> 6. Other lines should be formatted for readability, recognizing the existence of wide screens and scrollbars (sometimes a 200 character line is more readable, though rarely).

So I think yes we do for E501. Or at least, we encourage completely ignoring the rule in favor of "use whatever line length makes the code most readable".

E201 and E202 I think  more fall under tolerate.